### PR TITLE
feat: surface some globals around restarts for the manage command

### PR DESCRIPTION
**Pull request by mdevilliers**

### CODE REVIEW

File: internal/ctl/connectors/manage.go [hunk 1]
```diff
@@ -82,6 +90,18 @@ if you specify --once then it will sync once and then exit.`,
 	manageCmd.Flags().DurationVar(&params.HTTPClientTimeout, "http-client-timeout", params.HTTPClientTimeout, "HTTP client timeout")
 	_ = viper.BindPFlag("http-client-timeout", manageCmd.PersistentFlags().Lookup("http-client-timeout"))
 
+	manageCmd.Flags().IntVar(&params.GlobalMaxConnectorRestarts, "global-connector-max-restarts", params.GlobalMaxConnectorRestarts, "maximum times a failed connector will be restarted")
+	_ = viper.BindPFlag("global-connector-max-restarts", manageCmd.PersistentFlags().Lookup("global-connector-max-restarts"))
+
+	manageCmd.Flags().DurationVar(&params.GlobalConnectorRestartPeriod, "global-connector-restart-period", params.GlobalConnectorRestartPeriod, "period of time between failed connector restarts")
+	_ = viper.BindPFlag("global-connector-restart-period", manageCmd.PersistentFlags().Lookup("global-connector-restart-period"))
+
+	manageCmd.Flags().IntVar(&params.GlobalMaxTaskRestarts, "global-task-max-restarts", params.GlobalMaxTaskRestarts, "maximum times a failed task will be restarted")
```
Comment (the4thamigo-uk) [inline internal/ctl/connectors/manage.go]: max-task or task-max?

File: internal/ctl/connectors/manage.go [hunk 2]
```diff
@@ -82,6 +90,18 @@ if you specify --once then it will sync once and then exit.`,
 	manageCmd.Flags().DurationVar(&params.HTTPClientTimeout, "http-client-timeout", params.HTTPClientTimeout, "HTTP client timeout")
 	_ = viper.BindPFlag("http-client-timeout", manageCmd.PersistentFlags().Lookup("http-client-timeout"))
 
+	manageCmd.Flags().IntVar(&params.GlobalMaxConnectorRestarts, "global-connector-max-restarts", params.GlobalMaxConnectorRestarts, "maximum times a failed connector will be restarted")
```
Comment (the4thamigo-uk) [inline internal/ctl/connectors/manage.go]: maybe these strings could be `const` as they appear in a couple of places?

File: internal/ctl/connectors/manage.go (hunk 1 above)
Comment (mdevilliers) [reply internal/ctl/connectors/manage.go]: 👍

File: internal/ctl/connectors/manage.go (hunk 2 above)
Comment (mdevilliers) [reply internal/ctl/connectors/manage.go]: I don't think that would be an improvement.

Comment (the4thamigo-uk) [reply internal/ctl/connectors/manage.go]: Im talking about these strings `global-connector-max-restarts` etc... I think it is an improvement to move these into consts.

**Pull request merged**